docker/hyperdx/entry.local.base.sh

```diff
@@ -63,7 +63,7 @@ opampsupervisor --config /etc/otel/supervisor.yaml > /var/log/otel-collector.log
 
 # Start HyperDX app
 npx concurrently \
-  "--kill-others" \
+  "--kill-others-on-fail" \
```

Comment:
we want to use '--kill-others-on-fail' instead since the task can exit immediately